src/mmw/js/src/analyze/views.js

```diff
@@ -24,11 +23,29 @@ var AnalyzeWindow = Marionette.LayoutView.extend({
     regions: {
         headerRegion: '#analyze-header-region',
         detailsRegion: {
-            regionClass: TransitionRegion,
             el: '#analyze-details-region'
         }
     },
 
+    /*
+     Any idea why this doesn't work?
```

Comment:
It appears that items in the events hash only work on jquery events: http://backbonejs.org/#View-delegateEvents

Part of the reason this seems to not be functioning properly is that you are using the `trigger` method, which is a pure jQuery thing, whereas `triggerMethod` is the marionette method to trigger special event callbacks.